### PR TITLE
Refs #145 - Enable core_collect via playbook

### PR DESCRIPTION
**Pull request by apuntamb**

### CODE REVIEW

File: ansible-playbooks/ros_install_and_set_up.yml [hunk 1]
```diff
@@ -10,21 +10,18 @@
 - hosts: all
   gather_facts: True
   tasks:
-    - name: Install insights-client if not already installed.
+    - name: Install insights-client if not already installed
```
Comment (kgaikwad) [inline ansible-playbooks/ros_install_and_set_up.yml:13]:
```suggestion
    - name: Ensure latest insights-client is installed
```

File: ansible-playbooks/ros_install_and_set_up.yml [hunk 2]
```diff
@@ -10,21 +10,18 @@
...
       yum:
         name: insights-client
         state: latest
-    - name: Check if core_collect is enabled.
+    - name: Check if core_collect is enabled
+      shell: grep -c '^\s*core_collect\s*=\s*True\s*' /etc/insights-client/insights-client.conf || true
+      register: core_collect
+    - name: Enable core_collect if not already enabled
```
Comment (kgaikwad) [inline ansible-playbooks/ros_install_and_set_up.yml:20]:
```suggestion
    - name: Ensure core_collect is enabled
```

**Review by kgaikwad — CHANGES_REQUESTED** [review]
From @gnurag's [comment](https://github.com/RedHatInsights/ros-backend/pull/85#issuecomment-881231627), changes are working as expected.
Just added two small textual changes otherwise good to go :+1:

File: ansible-playbooks/ros_install_and_set_up.yml (hunk 1 above)
Comment (apuntamb) [reply ansible-playbooks/ros_install_and_set_up.yml:13]: Cool sure, I would prefer enhancing it bit more with:
`Ensure latest version of insights-client is installed`

**Review by kgaikwad — APPROVED** [review]

**Comment by kgaikwad** [comment]
Thanks @apuntamb & @gnurag for testing out the change.

**Pull request merged**